ui/src/api/patrons/patron.js

```diff
@@ -7,6 +7,8 @@ const listUrl = '/patrons/';
 // but not stored in the database. Instead we are using
 // `invenio_accounts_rest` users endpoint to retrieve
 // individual patron's information.
+
+// TODO: fix this
```

Comment:
I thought I was the only one leaving `TODOS` behind, and as @ntarocco wisely once said to me, fix it or remove it 🎉